web/app/themes/mitlib-parent/inc/content-pagefull.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * The template used for displaying page content in page.php
+ *
+ * @package MITlib_Parent
+ * @since 0.2.0
+ */
+
+namespace Mitlib\Parent;
+
+global $isRoot;
+?>		
+		
+		<div class="main-content content-main">
```

Comment:
Is the spacing of this intentionally like this?

Reply:
The spacing is likely an attempt to match indenting from whatever template is calling this. I don't agree with that goal, and tried to strip out extra indenting in some files that I touched a lot in this branch - but hit my limit with this ticket before I sat down and rationalized all the templates in this way.